src/components/SccRemoteImage.tsx

```
@@ -1,154 +1,167 @@
+    },
+    [imageUrl, measuredWidth],
+  );
 
+  // onLayout 후 사이즈가 있으면 ready
   useEffect(() => {
-    if (!imageLoading && imageHeight !== undefined) {
-      onReady?.();
-    }
-  }, [imageLoading, imageHeight, onReady]);
+    if (measuredWidth !== undefined && originalSize && !ready) {
 
```

ESLint

[error] Delete `⏎`

(prettier/prettier)